tests/conftest.py

```diff
@@ -0,0 +1,185 @@
+"""
+# run all tests:
+# pytest -sv --cov-report term-missing --cov=aeon_mecha -p no:warnings tests/dj_pipeline
+
+# run one test, debug:
+# pytest [above options] --pdb tests/dj_pipeline/test_ingestion.py -k <function_name>
+
+# run test on marker:
+# pytest -m <marker_name>
+"""
+
+import os
+import pathlib
+
+import datajoint as dj
+import pytest
+
+_tear_down = True  # always set to True since most fixtures are session-scoped
+_populate_settings = {"suppress_errors": True}
+
+
+def data_dir():
+    """
+    Returns test data directory
+    """
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
+
+
+@pytest.fixture(autouse=True, scope="session")
+def test_params():
+
+    return {
+        "start_ts": "2022-06-22 08:51:10",
+        "end_ts": "2022-06-22 14:00:00",
+        "experiment_name": "exp0.2-r0",
+        "raw_dir": "aeon/data/raw/AEON2/experiment0.2",
+        "qc_dir": "aeon/data/qc/AEON2/experiment0.2",
+        "test_dir": data_dir(),
+        "subject_count": 5,
+        "epoch_count": 1,
+        "chunk_count": 7,
+        "experiment_log_message_count": 0,
+        "subject_enter_exit_count": 0,
+        "subject_weight_time_count": 0,
+        "camera_qc_count": 40,
+        "camera_tracking_object_count": 5,
+    }
+
+
+@pytest.fixture(autouse=True, scope="session")
+def dj_config():
+    """If dj_local_config exists, load"""
+    dj_config_fp = pathlib.Path("dj_local_conf.json")
+    assert dj_config_fp.exists()
+    dj.config.load(dj_config_fp)
+    dj.config["safemode"] = False
+    assert "custom" in dj.config
+    dj.config["custom"][
+        "database.prefix"
+    ] = f"u_{dj.config['database.user']}_testsuite_"
+    return
```

Comment:
Can we drop the `return` statements in the fixtures that do not return anything?